tests/unit_tests/report/test_project_report.py

```diff
@@ -186,7 +186,7 @@ def test_display_model_performance_2(self, mock_logging):
             explainer=self.xpl,
             project_info_file=os.path.join(current_path, '../../data/metadata.yaml'),
             y_test=self.df['y'],
-            config=dict(metrics={'mse': 'sklearn.metrics.mean_squared_error'})
+            config=dict(metrics=[{'path': 'sklearn.metrics.mean_squared_error'}])
         )
         report.display_model_performance()
         self.assertEqual(mock_logging.call_count, 0)
```

Comment:
Can you add test with use of 'proba_values' ?

Reply:
Indeed, thank you for noticing, I added a test